File: src/main/java/Homework7/HotDrink.py
class HotDrink():
    def __init__(self, name, value, temperature):
        self.__temperature = temperature
        self.__value = value
        self.__name = name

    def getTemperature(self):
        return self.__temperature

    def setTemperature(self, temperature):
        self.__temperature = temperature

    def getValue(self):
        return self.__value

    def setValue(self, value):
        self.__value = value

    def getName(self):
        return self.__name

    def setName(self, name):
        self.__name = name

    def __str__(self):
        return f"Название: {self.__name}; Стоимость: {self.__value}; Температура: {self.__temperature}"

File: src/main/java/Homework7/HotDrink.java
package Homework4;

public class HotDrink {
    private int temperature;
    private int value;
    private String name;

    public HotDrink(String name, int value, int temperature) {
        this.name = name;
        this.value = value;
        this.temperature = temperature;
    }

    public int getTemperature() {
        return temperature;
    }

    public void setTemperature(int temperature) {
        this.temperature = temperature;
    }

    public float getValue() {
        return value;
    }

    public void setValue(int value) {
        this.value = value;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String toString() {
        return String.format("Название: %s; Стоимость: %d; Температура: %d", name, value, temperature);
    }
}